code/modules/mob/mob_helpers.dm

```diff
@@ -14,6 +14,49 @@
 	if(isStunned() || restrained())
 		return 1
 
+/mob/proc/get_screen_colour()
+	if(!client)
+		return 0
+	if(M_NOIR in mutations)
```

Comment:
Really, another loop through mutations every player tick?

Things like this is why life isn't fast you know.